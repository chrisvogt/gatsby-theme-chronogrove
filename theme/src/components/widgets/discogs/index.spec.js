import React from 'react'
import renderer from 'react-test-renderer'

import DiscogsWidget from './index'

// Mock the discogs widget
jest.mock('./discogs-widget', () => {
  return function MockDiscogsWidget() {
    return <div data-testid='discogs-widget'>Discogs Widget</div>
  }
})

describe('Discogs Index', () => {
  it('exports the discogs widget as default', () => {
    const tree = renderer.create(<DiscogsWidget />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
