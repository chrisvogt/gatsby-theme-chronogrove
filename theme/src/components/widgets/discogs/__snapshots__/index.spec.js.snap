// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Discogs Index exports the discogs widget as default 1`] = `
<div
  data-testid="discogs-widget"
>
  Discogs Widget
</div>
`;
